build-scripts/enable_derivatives.py

```diff
@@ -97,7 +97,8 @@ def main():
         raise RuntimeError("No Benchmark found!")
 
     for namespace, benchmark in benchmarks:
-        ssg.build_derivatives.profile_handling(benchmark, namespace)
+        if args[1] != "cs9":
+            ssg.build_derivatives.profile_handling(benchmark, namespace)
```

Comment:
It would probably need to add a comment explaining why is CentOS Stream 9 different than other derivatives.